model/config.go

```diff
@@ -1456,8 +1456,8 @@ func (s *FileSettings) SetDefaults(isUpdate bool) {
 		s.MaxFileSize = NewInt64(FileSettingsDefaultMaxFileSize) // 100MB (IEC)
 	}
 
-	if s.MaxVoiceMessagesFileSize == nil {
-		s.MaxVoiceMessagesFileSize = NewInt64(*s.MaxFileSize)
+	if s.MaxVoiceMessagesDuration == nil {
+		s.MaxVoiceMessagesDuration = NewInt64(0)
```

Comment:
Same comment I left on the webapp side. Maybe have a sensible default (e.g. a few minutes) since unlimited in production is usually never a good idea.

Reply:
I've set this to 5min